topics/firefox/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: firefox, seamonkey, waterfox, palemoon, icecat, iceweasel, torbrowser
+created_by: Mozilla
+display_name: Mozilla Firefox
+github_url: https://github.com/mozilla
+released: '2002-09-23'
+short_description: An advanced free open-source web browser
+topic: firefox
+url: https://mozilla.org/firefox
+wikipedia_url: https://en.wikipedia.org/wiki/Firefox
+related: WebExtension
+---
+A web browser by Mozilla which once was the most hackable and had the most powerful extensions.
```

Comment:
`A web browser by Mozilla which is hackable and has powerful extensions.

Reply:
It's not anymore.

Reply:
@KOLANICH Even if that's the case it's a bit negative/passive aggressive to reference this in the past tense so can you try to reword this to talk about the current state of Firefox rather than what it used to be. How about `Mozilla Firefox is a fast, cross-platform web browser developed by the Mozilla Foundation.`?

Reply:
@KOLANICH Can you update this so there is no past tense in the description, please? Thanks!

Reply:
>How about Mozilla Firefox is a fast, cross-platform web browser developed by the Mozilla Foundation.?

Not quite good. There are some legacy (now - obsolete and deprecated) extensions on GitHub. This tag is needed only for them, because WebExtension API is crossbrowser. So I guess we need to put there some clue why these extensions don't work anymore and won't work. Maybe we don't need this tag at all, since the API is deprecated and dropped, these extensions have no future. Al least if someone not forked Firefox and started maintaining the fork, fixing xpcom to becompatible with the latest mozilla code base, building builds and resolving merge conflicts.